public/app/features/transformers/extractFields/ExtractFieldsTransformerEditor.tsx

```diff
@@ -96,6 +105,13 @@ export const extractFieldsTransformerEditor = ({
           />
         </InlineField>
       </InlineFieldRow>
+      {options.format === 'regexp' && (
+        <InlineFieldRow>
+          <InlineField label="RegExp">
+            <Input placeholder="/(?<NewField>.*)/" value={options.regExp} onChange={onRegexpChange} />
```

Comment:
Could we make InlineField have `interactive={true}` with that placeholder as a tooltip? Nothing fancy, just with the text so someone could copy/paste it as a starter template. Like https://github.com/grafana/grafana/blob/main/public/app/features/transformers/editors/FormatTimeTransformerEditor.tsx#L85

Reply:
:thinking: